src/lib/chat/matrix/utils.test.ts

```diff
@@ -0,0 +1,78 @@
+// Import your function and any necessary dependencies
+import { getFilteredMembersForAutoComplete } from './utils';
+
+// Example room members data
+const roomMembers = [
+  {
+    userId: '@domw:zero-synapse-development.zer0.io',
+    matrixId: '@domw:zero-synapse-development.zer0.io',
+    firstName: 'domw',
+    lastName: '',
+    profileId: '',
+    isOnline: false,
+    profileImage: '',
+    lastSeenAt: '',
```

Comment:
Do any of these values impact the tests? If not, just remove them.

Reply:
removed